# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/picture_graph.rs
# Commit: 2b42abbdb0df
# Full Hash: 2b42abbdb0df38f31dfa1178fe3b5f773f8e4812
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-20 21:58:21
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 2 - Performance and quality fixes for part 1. r=gfx-reviewers,nical
#   
#   * Add support for local scale factors to a surface, allowing it to
#     be rasterized in root coordinate space. This allows snapping to
#     work across surfaces where the surface transform is a fractional
# ==============================================================================

diff -r 9fa6f71111d0 -r 2b42abbdb0df gfx/wr/webrender/src/picture_graph.rs
--- a/gfx/wr/webrender/src/picture_graph.rs	Sun Feb 20 18:56:54 2022 +0000
+++ b/gfx/wr/webrender/src/picture_graph.rs	Sun Feb 20 18:56:55 2022 +0000
@@ -108,6 +108,7 @@
 
                 match pictures[pic_index.0].assign_surface(
                     frame_context,
+                    parent_surface_index,
                     tile_caches,
                     surfaces,
                 ) {